azure-sql/database/resource-health-to-troubleshoot-connectivity.md

```diff
@@ -51,6 +55,10 @@ The health status of **Unknown** indicates that **Resource health** hasn't recei
 
 :::image type="content" source="./media/resource-health-to-troubleshoot-connectivity/sql-resource-health-unknown.jpg" alt-text="A screenshot of the Azure portal showing the status message for the state of Unknown.":::
 
+## Alert Time
+
+The time shown by the **Resource health** alert will not line up with the times of the login failures that caused the alert. This is because it takes several minutes for the telemetry to be collected and analyzed, to determine that there is a **Resource health** issue. So, the time indicated in the **Resource health** alert will be several minutes after the login failures.
```

Comment:
IIRC, this would be a relevant statement to add here. If so, can you add?

>Additionally, the time interval when the login failures were occurring might often be shorter than the time interval in the resource health alert.